tests/Set/Symfony61/Fixture/set_description_at_top.php.inc

```diff
@@ -0,0 +1,57 @@
+<?php
+
+namespace Rector\Symfony\Tests\Set\Symfony61\Fixture;
+
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Output\OutputInterface;
+
+class SetDescriptionAtTop extends \Symfony\Component\Console\Command\Command
+{
+    public function configure()
+    {
+        $this
+            ->setName('some:command')
+            ->setDescription('Description');
+    }
+
+    protected function initialize(InputInterface $input, OutputInterface $output): void
+    {
+        parent::initialize($input, $output);
+    }
+
+     protected function execute(InputInterface $input, OutputInterface $output): int
+    {
+        return 0;
+    }
+}
+
+?>
+-----
+<?php
+
+namespace Rector\Symfony\Tests\Set\Symfony61\Fixture;
+
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Output\OutputInterface;
+
+class SetDescriptionAtTop extends \Symfony\Component\Console\Command\Command
+{
+    protected static $defaultDescription = 'Description';
```

Comment:
Why add this test for 6.1? It is just a copy of one of the `CommandDescriptionToPropertyRector` tests

Reply:
Okay so i added `CommandPropertyToAttributeRector` should i add a fixture of that test or should this be skipped as this is tested elsewhere?

Reply:
If you ask me, the expected refactor for this deprecation should be a static property to an attribute.

Using `setName()` and `setDescription()` is not deprecated. The Symfony documentation just prefers it the "new" way for optimizations:

> Defining the $defaultDescription static property instead of using the setDescription() method allows to get the command description without instantiating its class. This makes the php bin/console list command run much faster.

https://symfony.com/doc/current/console.html#configuring-the-command

So my personal expectation for using this set would be a refactors like this:

```diff
-#[\Symfony\Component\Console\Attribute\AsCommand(name: 'some:command']
+#[\Symfony\Component\Console\Attribute\AsCommand(name: 'some:command', description: 'Description')]
class SetDescriptionAtTop extends \Symfony\Component\Console\Command\Command
{
-    protected static $defaultDescription = 'Description';
}
```
and
```diff
+#[\Symfony\Component\Console\Attribute\AsCommand(name: 'some:command', description: 'Description')]
class SetDescriptionAtTop extends \Symfony\Component\Console\Command\Command
{
-    protected static $defaultName = 'some:command';
-    protected static $defaultDescription = 'Description';
}
```
But that last example has already been covered by the tests for `CommandPropertyToAttributeRector`, however we want to test changes using `SymfonySetList::SYMFONY_61`.

Maybe @TomasVotruba can guide us here. Do we need duplicate tests when a Rector is used in a set to verify behaviour of the set?

Reply:
> Maybe @TomasVotruba can guide us here. Do we need duplicate tests when a Rector is used in a set to verify behaviour of the set?


That's not necessary indeed. But the bug should be fixed, so CI pass.

Reply:
Check i added missing stubs